cookbooks/aws-parallelcluster-config/recipes/cloudwatch_agent.rb

```diff
@@ -64,18 +64,24 @@
   command "#{node.default['cluster']['cookbook_virtualenv_path']}/bin/python #{validator_script_path}"
 end
 
+fetch_config 'Fetch and load cluster configs' unless node['cluster']['scheduler'] == 'awsbatch'
+
+load_cluster_config
+
 execute "cloudwatch-config-creation" do
   user 'root'
   environment(
     'LOG_GROUP_NAME' => node['cluster']['log_group_name'],
     'SCHEDULER' => node['cluster']['scheduler'],
     'NODE_ROLE' => node['cluster']['node_type'],
-    'CONFIG_DATA_PATH' => config_data_path
+    'CONFIG_DATA_PATH' => config_data_path,
+    'CLUSTER_CONFIG_PATH' => node['cluster']['cluster_config_path']
```

Comment:
node['cluster']['cluster_config_path'] does not exist if scheduler is awsbatch

Reply:
Good catch! 
Remove it from environment() and set `cluster_config_path = node['cluster']['scheduler'] == 'plugin' ? "--cluster-config-path #{node['cluster']['cluster_config_path']}" : ""`, only pass `--cluster-config-path` when there's scheduler plugin.